courses/pyladies/plan.yml

```diff
@@ -1,67 +1,99 @@
 - name: Instalace
   materials:
-  - link: lessons/beginners/cmdline
-  - link: lessons/beginners/install
-  - link: lessons/beginners/install_editor
-  - link: lessons/beginners/install_git
+  - name: Úvod do příkazové řádky
```

Comment:
It *is* nice. External links should be an exception, in the long run cheatsheets and handouts would all be on naucse. We'll always need a few external links, but that shouldn't be a reason to duplicate all the names (and increase the chance for copy/paste errors) whenever a new course is created.